content/docs/14-troubleshooting/30-pcg.md

```diff
@@ -14,14 +14,15 @@ import InfoBox from 'shared/components/InfoBox';
 
 # Private Cloud Gateway (PCG)
 
-The following are scenarios you may encounter when deploying and using a Private Cloud Gateway.
+You may encounter one of the following scenarios when deploying and using a Private Cloud Gateway in your data center. Some of the scenarios below apply to any data center, whereas others apply to specific data centers, such as VMware. Each scenario covers a specific problem, including an overview of the issue and its possible causes, followed by a detailed set of debug steps to help identify and resolve the underlying problem.  
```

Comment:
```suggestion
When deploying and using a Private Cloud Gateway in your computing environment, you may encounter one of the following scenarios. Some of the scenarios below apply to all compute environments, whereas others apply to specific self-hosted environments, such as VMware. Each scenario covers a specific problem, including an overview of the issue and its possible causes, followed by a detailed set of debug steps to help identify and resolve the underlying problem.  
```
I touched up the intro and made it clearer that it's not always self-hosted environments.